swagger.yaml

```diff
@@ -5655,6 +5675,20 @@ definitions:
       role:
         $ref: "#/definitions/disk-role"
 
+  disk-skip-formatting-params:
+    type: object
+    description: Allows an addition or removal of a host disk from the host's skip_formatting_disks list
+    required:
+      - 'disk_id'
+      - 'skip_formatting'
```

Comment:
@omertuc why do you need this boolean flag? Isn't the presence of the disk-id in the disks-skip-formatting list enough?

Reply:
The API isn't to provide all the disks to skip formatting each time, and that replaces the previous list.  If a disk was previously selected to be skipped, you undo it by passing skip_formatting=false next time.
If you pass skip_formatting=true for a disk and it was already set to be skipped, or pass false and it was already false, then the API will accept it.

Reply:
OK now I get it, 10x